webapp/views/wpt-results.js

```diff
@@ -897,6 +929,92 @@ class WPTResults extends AmendMetadataMixin(Pluralizer(WPTColors(WPTFlags(PathIn
     this.path = this.searchResults[(n + next) % n].test;
   }
 
+  sortTestName() {
+    if (!this.displayedNodes) {
+      return;
+    }
+
+    this.isPathSorted = !this.isPathSorted;
+    this.sortRow = new Array(this.testRuns.length).fill(false);
+    const sortedNodes = this.displayedNodes.slice();
+    const self = this;
+    sortedNodes.sort(function (a, b) {
+      if (self.isPathSorted) {
+        return self.compareTestNameDefaultOrder(a, b);
+      }
+      return self.compareTestNameDefaultOrder(b, a);
+    });
+    this.displayedNodes = sortedNodes;
+  }
+
+  compareTestName(a, b) {
+    if (this.isPathSorted) {
+      return this.compareTestNameDefaultOrder(a, b);
+    }
+    return this.compareTestNameDefaultOrder(b, a);
+  }
+
+  compareTestNameDefaultOrder(a, b) {
+    const pathA = a.path.toLowerCase();
+    const pathB = b.path.toLowerCase();
+    if (pathA < pathB) {
+      return -1;
+    }
+
+    if (pathA > pathB) {
+      return 1;
+    }
+    return 0;
+  }
+
+  sortTestResults() {
+    const [index] = arguments;
```

Comment:
Could `index` just be passed into the function as a param?

Reply:
Done